build.gradle

```diff
@@ -16,20 +16,20 @@ dependencies {
         	'com.hcl:appscan.sdk:1.0.9'
 }
 
-group = 'com.ibm.security'
-version = '1.0.2'
+group = 'com.hcl.security'
+version = '1.0.0'
 
 pluginBundle {
-	website = 'https://www.ibm.com/us-en/marketplace/application-security-on-cloud'
+	website = 'https://www.ibm.com/support/knowledgecenter/en/SSYJJF_1.0.0/ApplicationSecurityonCloud/src_irx_gen_gradle.html'
 	vcsUrl = 'https://github.com/hclproducts/application-security-gradle-plugin'
```

Comment:
Can you update this link to:
https://help.hcltechsw.com/appscan/ASoC/src_irx_gen_gradle.html

Reply:
@mattmurp , suggested change has been committed.